UI/cypress/e2e/results.feature

```diff
@@ -112,4 +112,22 @@ Feature: User is shown search results
   Scenario: Local Education Authority name is displayed for postcode
     Given a user has arrived on the 'Search results' page for 'Key stage 2 Maths' for postcode 'HP4 3LG'
     Then they will see the results summary for 'Hertfordshire'
-  
\ No newline at end of file
+  
+  Scenario: Display ‘clear filters; button when 'Search results' page is displayed
+    Given a user has arrived on the 'Search results' page
+    Then the ‘clear filters’ button is displayed at the bottom of the filters section
+
+  Scenario: Nothing is selected if when clear all filters is clicked
+    Given a user has arrived on the 'Search results' page
+    When the ‘clear filters’ button as been selected
+    Then no subject should be shown as selected
+    And they will see the tuition type 'Any' is selected
+
+   Scenario: All subject filters are collapsed when clear all filters is clicked
+    Given a user has arrived on the 'Search results' page
+    When the ‘clear filters’ button as been selected
+    Then all subject filters are collapsed
+
+  Scenario: Results are updated when clear all filters is clicked
+    Given a user has arrived on the 'Search results' page
+    Then results are updated after filters are cleared
```

Comment:
I think there is a missing scenario here which is also not in the ACs. Something like:

Scenario: Postcode is retained when clear all filters is clicked

Reply:
Added new test